cfpb/consumerfinance.gov
# Prepaid agreements: Move search.less into prepaid agreements app

This CSS is only used by the prepaid agreements search app.

## Changes

- Move search.less to prepaid agreements app


## How to test this PR

1. `yarn build` and http://localhost:8000/data-research/prepaid-accounts/search-agreements/ should be unchanged.
